src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -93,14 +80,15 @@ impl fmt::Display for VmConfig {
     }
 }
 
-/// Spec for partial configuration of the machine.
-/// This struct mirrors all the fields in `VmConfig`.
+/// Struct used in PATCH `/machine-config` API call.
+/// Used to update `VmConfig` in `VmResources`.
+/// This struct mirrors all the fields in `MachineConfig`.
 /// All fields are optional, but at least one needs to be specified.
 /// If a field is `Some(value)` then we assume an update is requested
 /// for that field.
 #[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
-pub struct VmUpdateConfig {
+pub struct MachineConfigUpdate {
```

Comment:
nit: It seems that there are some comments include `VmUpdateConfig`. Could you update those comments as well?

Reply:
fixed